File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] res = new int[2*n];
        int j = 0;
        for(int i = 0; i < nums.length; i+=2){
            res[i] = nums[j++];
        }
        for(int i = 1; i < nums.length; i+=2){
            res[i] = nums[j++];
        }
        return res;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        for(int i = 0; i < n; i++){
            nums[i + n] += nums[i] * 10000;
        }
        for(int i = 0; i < n; i++){
            nums[2 * i] = nums[i + n]/10000;
            nums[(2 * i) + 1] = nums[i + n] % 10000;
        }
        return nums;
    }
};